includes/Captcha/Providers/CloudflareTurnstileProvider.php

```
@@ -0,0 +1,158 @@
+    }
+
+    /**
+     * Register and enqueue Turnstile API and helper script.
+     *
+     * @return void
+     */
+	public function register_assets(): void {
+        if ( ! $this->is_ready() ) {
+            return;
+        }
+        $site = $this->get_option( 'turnstile_site_key', '' );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$site'. (undefined)

(UnusedLocalVariable)